test/e2e/specs-gutenberg/wp-calypso-gutenberg-focused-launch-spec.js

```diff
@@ -229,6 +230,38 @@ describe( `[${ host }] Calypso Gutenberg Editor: Focused launch on (${ screenSiz
 			assert( selectedPlanIsPersonalMonthlyPlan, 'The personal monthly plan was not selected.' );
 		} );
 
+		step( 'Can reload block editor and reopen focused launch', async function () {
+			// Reload block editor
+			await driver.navigate().refresh();
+
+			// Press "Reload" on confirmation dialog when block editor asks if user really wants to navigate away.
+			try {
+				await driver.wait( until.alertIsPresent(), 4000 );
+				const alert = await driver.switchTo().alert();
+				await alert.accept();
+			} catch ( e ) {
+				// This doesn't happen when autosave hasn't kicked in so
+				// if driver.wait throws and error we catch it here to allow
+				// the step to continue running.
+			}
+
+			// Wait for block editor to load and switch frame context to block editor
+			await GutenbergEditorComponent.Expect( driver );
+
+			// Click on the launch button
+			const launchButtonSelector = By.css( '.editor-gutenberg-launch__launch-button' );
```

Comment:
Maybe a bit of a nitpick, but it's possible to select the element with `driverHelper.getElementByText()` too. Since it resembles the user's behavior better, should we implement it?

Reply:
Technically, we could still do `getElementByText(driver, '.editor-gutenberg-launch__launch-button', 'Launch')`.

If we keep the classname, this would just reinforce the fact that the launch button has to have the label "Launch".

If we lose the classname, could be potentially dangerous if there are other buttons with the "Launch" label on it.

Hmm, let's keep it as it is for now. :)

Reply:
> [...] could be potentially dangerous if there are other buttons with the "Launch" label on it.

When this happens, I expect our tests to fail, since it changes the app's UI/UX considerably and we should be notified/warned about that. 

Besides that, `getElementByText(driver, '.editor-gutenberg-launch__launch-button', 'Launch')` resembles the user's behavior best, since the user is looking for a button with the "Launch" label and it's not looking for a CSS class name. Therefore it generates the maximum amount of code confidence.

Reply:
Haha alright! Let's do it! https://github.com/Automattic/wp-calypso/pull/51846/commits/ee945cf4900b5af56881a1685570850e35d843dc

Reply:
Thanks!